### ISSUE TITLE
Setting ip_check_disabled has no effect

### ISSUE THREAD

**Issue by bakern** [unixsurfer/anycast_healthchecker#50]
Hello,

Thank you for creating this tool.  While testing, it seems like setting `ip_check_disabled` to true does not have the desired effect.  Sample logs:

```
DEBUG    DNS      ip_check_disabled=True:<class 'bool'>
INFO     DNS      checking for IP assignment on interface ens33 is disabled
DEBUG    DNS      running /sbin/ip address show dev ens33 to 10.1.1.12/32
WARNING  DNS      10.1.1.12/32 isn't assigned to ens33 interface
WARNING  DNS      status DOWN because 10.1.1.12/32 isn't assigned to ens33 interface.
```

I see this code block that checks to see if it is disabled, but it seems to continue on checking anyway after setting `result = True`:
https://github.com/unixsurfer/anycast_healthchecker/blob/5ec2030c17137569e8b527aaabc486c1e7ad4cb4/anycast_healthchecker/servicecheck.py#L163-L166

Please let me know if you can confirm this, or if I'm misunderstanding.

Thanks!

**Comment by unixsurfer** [comment]
Hi @bakern,

Thank you so much for reporting this issue, and I apologize for the bug! I’ve [fixed](#51) it and will be making a release soon to resolve it.

Thanks again for your help!

**Comment by bakern** [comment]
Hi @unixsurfer,

Thanks for fixing this, I appreciate it!

I noticed that you tagged a 0.9.9 release but Github and PyPI still show 0.9.8 as the latest.  Unless you are waiting for a while to verify, would you be able to set 0.9.9 as the latest release?

Thanks!
